PukingPredator/Assets/Scripts/Physics/CollapseIfHeavyPlayer.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CollapseIfHeavyPlayer : MonoBehaviour
+{
+    // Put on tiles with PhysicsBehavior and adjust the massThreshold
```

Comment:
Remove the comment and just make it a requirement of the component (by adding "[Require...]" on the line before "public class ..."). It should be strictly enforced if it's a requirement, not left as a comment that people can't see without looking inside the class.